qa-tests-backend/src/test/groovy/ExternalNetworkSourcesTest.groovy

```diff
@@ -244,15 +223,29 @@ class ExternalNetworkSourcesTest extends BaseSpecification {
     private static deleteNetworkEntity(String entityID) {
         // Use network graph client without the wrapper because we need the test to fail if the deletion fails.
         NetworkGraphService.getNetworkGraphClient()
-            .deleteExternalNetworkEntity(Common.ResourceByID.newBuilder().setId(entityID).build())
+                .deleteExternalNetworkEntity(Common.ResourceByID.newBuilder().setId(entityID).build())
+    }
+
+    private static void verifyEdge(String deploymentUid, String subnetID, int sinceSeconds = 0, int retries = 4) {
+        Helpers.withRetry(retries, 30) {
+            assert NetworkGraphUtil.checkForEdge(deploymentUid, subnetID, since(sinceSeconds), 180)
+        }
```

Comment:
Not sure whether the default value for `sinceSeconds` makes sense here. Can we request a network graph representing the last 0 seconds?

Reply:
There is a special case when `sinceSeconds <= 0` then we set since to `null`. If you prefer I can change `int` to `Integrer` and set `null` instead of 0

Reply:
oh, so the value of 0 would trigger some internal default. This is okay then